service/src/main/kotlin/fi/espoo/evaka/application/ApplicationQueries.kt

```diff
@@ -249,6 +250,9 @@ fun fetchApplicationSummaries(
         JOIN care_area ca on d.care_area_id = ca.id
         JOIN person child ON child.id = a.child_id
         LEFT JOIN placement_plan pp ON pp.application_id = a.id AND a.status = 'WAITING_UNIT_CONFIRMATION'::application_status_type
+        LEFT JOIN  (
+            SELECT placement_plan.*, daycare.name unit_name FROM placement_plan JOIN daycare ON placement_plan.unit_id = daycare.id
+        ) pp2 ON pp2.application_id = a.id
```

Comment:
Seems kind of silly to join twice on `placement_plan` just because some of its fields are interesting only when application has some status.

Reply:
Yep that's kinda sad but I'm not quite sure how to overcome that. If it so that those other fields are needed only when application has that specific status.